scubainit/src/main.rs

```diff
@@ -368,3 +370,20 @@ fn setup_logging() -> Result<()> {
         .verbosity(verbosity)
         .init()?)
 }
+
+// Restore the default SIGPIPE action.
+// Rust pre-main code may change the SIGPIPE disposition to ignore:
+// https://github.com/rust-lang/rust/issues/62569
+// https://github.com/rust-lang/rust/issues/97889
```

Comment:
I like the detail in this comment. I might add a note that `scubainit` doesn't rely on any specific `SIGPIPE` behavior, but that signal dispositions are inherited through `execve` and that Rust's pre-main code shouldn't influence the process executed by scuba.

From `man signal(7)`
> Signal dispositions
> ...
> A child created via fork(2) inherits a copy of its parent's signal  dispositions.   During  an
> execve(2),  the  dispositions of handled signals are reset to the default; the dispositions of
> ignored signals are left unchanged. 

The comment should help to point out that this code may be unnecessary in the future if Rust's `execve` library function is changed to restore `SIGPIPE`.

Reply:
Thanks. You're right I kind of glossed over why we care. Updated the comment (mostly verbatim from your comment).